dev_utils/docker-compose.yml

```
@@ -115,22 +128,37 @@ services:
       - "8000:8000"
       - "8001:8001"
 
+  database:
+    container_name: db
+    image: neicnordic/sda-db:v2.0.0
+    environment:
+      - DB_LEGA_IN_PASSWORD=inpass
+      - DB_LEGA_OUT_PASSWORD=outpass
```

```suggestion
      - DB_LEGA_IN_PASSWORD=lega_in
      - DB_LEGA_OUT_PASSWORD=lega_out
```
Just to keep them the same in every repo